runtests.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2009- The Spyder Development Team
+# Licensed under the terms of the MIT License
+#
+
+"""
+"""
+
+# Third party imports
+import qtpy
+import pytest
+
+
+def main():
+    pytest.main(['-x', 'spyderlib',  '-v', '-rw', '--durations=10',
+                 '--cov=spyderlib', '--cov-report=term-missing'])
```

Comment:
I would like to add more stuff here, like flake8 checks, yapf and stuff, so having a separate script is kinda nicer.

That is why I chose to use it like this.